### PR TITLE
Docs: New landing page design

### PR DESCRIPTION
**Pull request by hdbv95**
New landing design
Resolves [#959](https://github.com/redpanda-data/documentation/issues/959)

### CODE REVIEW

File: docs/home/platform/platform.js [hunk 1]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
   },
 ];
 
+const FooterMenuList = [
+  {
+    title: 'Get Started',
+    items: [
+      { text: 'Get started with Redpanda', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:74]:
```suggestion
      { text: 'Introduction to Redpanda', url: "" },
```

File: docs/home/platform/platform.js [hunk 2]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+      { text: 'Deploying with RedPanda', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:75]:
```suggestion
      { text: 'How Redpanda Works', url: "" },
```

File: docs/home/platform/platform.js [hunk 3]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+      { text: 'Redpanda Console', url: "" },
+      { text: 'rpk Command Reference', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:77]:
```suggestion
      { text: 'Redpanda Quickstart', url: "" },
```

File: docs/home/platform/platform.js [hunk 4]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+      { text: 'Tiered Storage', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:78]:
```suggestion
      { text: 'Install rpk', url: "" },
```

File: docs/home/platform/platform.js [hunk 5]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+  },
+  {
+    title: 'Develop',
+    items: [
+      { text: 'Get started with Redpanda', url: "" },
+      { text: 'Deploying with RedPanda', url: "" },
+      { text: 'Redpanda Console', url: "" },
+      { text: 'rpk Command Reference', url: "" },
+      { text: 'Tiered Storage', url: "" },
+      { text: 'Sizign Guidelines', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:90]:
```suggestion
      { text: 'Supported Kafka Clients', url: "" },
      { text: 'Set Up and Use Redpanda with Kafka client libraries', url: "" },
      { text: 'Set Up and Use Redpanda with NodeJS', url: "" },
      { text: 'Set Up and Use Redpanda with HTTP Proxy API', url: "" },
      { text: 'Designing the Data Stream', url: "" },
      { text: 'Producing Data', url: "" },
      { text: 'Consuming Data', url: "" },
      { text: 'Transactions', url: "" },
```

File: docs/home/platform/platform.js [hunk 6]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+  },
+  {
+    title: 'Deploy',
+    items: [
+      { text: 'Get started with Redpanda', url: "" },
+      { text: 'Deploying with RedPanda', url: "" },
+      { text: 'Redpanda Console', url: "" },
+      { text: 'rpk Command Reference', url: "" },
+      { text: 'Tiered Storage', url: "" },
+      { text: 'Sizign Guidelines', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:101]:
```suggestion
      { text: 'Choosing a Deployment Option', url: "" },
      { text: 'Explore RedPanda Features', url: "" },
      { text: 'Cloud Deployment', url: "" },
      { text: 'Self-Hosted Deployment', url: "" },
```

File: docs/home/platform/platform.js [hunk 7]
```diff
@@ -66,14 +67,66 @@ const FeatureList = [
...
+  },
+  {
+    title: 'Manage',
+    items: [
+      { text: 'Get started with Redpanda', url: "" },
+      { text: 'Deploying with RedPanda', url: "" },
+      { text: 'Redpanda Console', url: "" },
+      { text: 'rpk Command Reference', url: "" },
+      { text: 'Tiered Storage', url: "" },
+      { text: 'Sizign Guidelines', url: "" },
```
Comment (Feediver1) [inline docs/home/platform/platform.js:112]:
```suggestion
      { text: 'Manage Cloud', url: "" },
      { text: 'Managed Self Hosted', url: "" },
      { text: 'Manage RedPanda on Kubernetes', url: "" },
      { text: 'Manage Redpanda Console', url: "" },
      { text: 'Security', url: "" },
      { text: 'Migration', url: "" },
```

**Comment by micheleRP** [comment]
@Feediver1 What do you think about having Learn more links instead of text on the landing page tiles? (i.e., Learn more instead of "Try out Redpanda" and "Browse all commands for running Redpanda" and "Size your production clusters to maximize performance")

**Comment by micheleRP** [comment]
@Feediver1 What do you think about replacing "Set Up and Use" with "Run" in the following: 
- Set Up and Use Redpanda with Kafka client libraries
- Set Up and Use Redpanda with NodeJS
- Set Up and Use Redpanda with HTTP Proxy API

**Comment by micheleRP** [comment]
Our page titles aren't consistent right now, but at least on the landing page we could use similar structure for everything in the lists; e.g., change Producing to Produce, Choosing to Choose, etc. (to be consistent with Install, Manage, Explore).

**Comment by hdbv95** [comment]
Closing. changes already at sidebar [PR](https://github.com/redpanda-data/documentation/pull/972)

**Pull request closed without merging**